mlflow/resource_experiment.go

```diff
@@ -8,15 +8,15 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-// MLFLowExperiment defines the parameters that can be set in the resource.
-type MLFLowExperiment struct {
+// MLFlowExperiment defines the parameters that can be set in the resource.
+type MLFlowExperiment struct {
 	Name        string `json:"name"`
-	Tags        []Tag  `json:"tags,omitempty"`
+	Tags        []Tag  `json:"tags,omitempty" tf:"force_new"`
 	Description string `json:"description,omitempty"`
 }
 
-// MLFLowExperimentAPI defines the response object from the API
-type MLFLowExperimentAPI struct {
+// MLFlowExperimentAPI defines the response object from the API
+type MLFlowExperimentAPI struct {
 	ExperimentId     string `json:"experiment_id"`
 	Name             string `json:"name"`
 	ArtifactLocation string `json:"artifact_location,omitempty"`
```

Comment:
Should we make artifact location available within terraform, I could see where it might be useful as an input into something else?